### PR TITLE
Update LaCE-onboarding-guide.md

### LINKED ISSUE #156: Update LaCE-onboarding-guide.md
The aim of this issue is to update the LaCE onboarding guide wiki page so that it can be used as a reference by new contributors who are working on LaCE issues. The specific things to do are:

1. Fix spelling errors noted in the review: https://github.com/oppia/oppia-web-developer-docs/pull/144/files
2. Add images/screenshots to illustrate key steps
3. Add an overview section (suggested by @Nik-09 in https://github.com/oppia/oppia-web-developer-docs/issues/28#issuecomment-1507114390)

### PR DESCRIPTION
**Pull request by prafulbbandre**
added images

### CODE REVIEW

**Comment by gp201** [comment]
@prafulbbandre is this PR ready to be reviewed?

**Comment by prafulbbandre** [comment]
@gp201 i am going to add more info(screenshot) to it. I will let you know when it's ready.

**Review by seanlip — APPROVED** [review]
Thanks @prafulbbandre! No concerns from my end.

File: LaCE-onboarding-guide.md [hunk 1]
```diff
@@ -37,31 +37,60 @@ This page will help you solve common queries that you may face while working on
...
-- To generate data, we must first navigate to the admin page. We will automatically be redirected to the `ACTIVITIES` tab.
+- To generate data, we must first navigate to the admin page.
+
+![Admin Page](https://github.com/prafulbbandre/oppia-web-developer-docs/assets/109658545/bbfb7acd-356f-41a1-8f4e-f7747a9de805)
+
+-  We will automatically be redirected to the `ACTIVITIES` tab.
 - This tab will have several sections:
   - Reload a single exploration: This section has preloaded explorations. These can be loaded onto your local server by clicking the `Reload` button for each of them.
+
+![Activities Tab](https://github.com/prafulbbandre/oppia-web-developer-docs/assets/109658545/39a77f42-c38b-4f5b-9bd9-29366ab8bcb8)
```
Comment (gp201) [inline LaCE-onboarding-guide.md:53]: All the images seems to be linked to your fork. Can you move them to the oppia/oppia-web-developer-docs repo?

**Review by gp201 — CHANGES_REQUESTED** [review]
Thanks, @prafulbbandre ! I left some comments. PTAL

**Comment by gp201** [comment]
Can you combine #173 with #170 ?

**Comment by prafulbbandre** [comment]
It's combined in #173

**Pull request closed without merging**